types/react-native-signature-capture/index.d.ts

```diff
@@ -56,6 +56,13 @@ export interface SignatureCaptureProps extends ViewProps {
      */
     maxSize?: number | undefined;
 
+    /**
+     * Sets the color of the signature. Defaults to black.
+     *
+     * @default #000000
```

Comment:
Nitpicky, but this should probably be surrounded in quotes since it's a string:

```suggestion
     * @default "#000000"
```